src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -230,6 +230,17 @@ public function testLoadAnonymousServices()
         $this->assertSame($fooArgs[0], $barArgs[0]);
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Top-level anonymous services are deprecated since Symfony 3.4, the "id" attribute will be required in version 4.0 in %sservices_without_id.xml.
```

Comment:
it includes an absolute path, so this deals with phpunit's `assertStringMatchesFormat`